README.md

```diff
@@ -62,7 +62,22 @@ const options = {
 }
 
 ```
+### Using @fastify/env to configure other plugins
+The `@fastify/env` plugin loads asynchronously. If you wish to use its values in a different plugin before the boot sequence, you need to make sure that:
+1. `@fastify/env` is registered first.
+2. Await the plugin registration or await after()
 
+```js
+await fastify.register(fastifyEnv)
+// fastify.config can be used in here
+```
+
+OR
+```js
+fastify.register(fastifyEnv)
+await fastify.after()
```

Comment:
```suggestion
await fastify
```

Reply:
Works for me, I was just following the original comment though.